src/main/java/io/quarkus/activity/model/GitHubActivities.java

```diff
@@ -1,22 +1,32 @@
 package io.quarkus.activity.model;
 
+import static java.util.stream.Collectors.toList;
+
 import io.quarkus.runtime.annotations.RegisterForReflection;
 
 import java.util.List;
+import java.util.function.Predicate;
 
 @RegisterForReflection
 public class GitHubActivities {
+
+    private static final String OPEN_STATE = "OPEN";
+
     public User user;
     public List<ActivityEntry> pullRequestsActivities;
     public List<ActivityEntry> issuesActivities;
     public List<ActivityEntry> issueCommentsActivities;
+    public List<ActivityEntry> openPullRequestsActivities;
+    public List<ActivityEntry> openIssuesActivities;
 
     public GitHubActivities(User user, List<ActivityEntry> pullRequestsActivities, List<ActivityEntry> issuesActivities,
                             List<ActivityEntry> issueCommentsActivities) {
         this.user = user;
         this.pullRequestsActivities = pullRequestsActivities;
         this.issuesActivities = issuesActivities;
         this.issueCommentsActivities = issueCommentsActivities;
+        this.openPullRequestsActivities = pullRequestsActivities.stream().filter(hasOpenState()).collect(toList());
```

Comment:
Do we need whole collection? Seems that just `int` for size would work just fine.

Reply:
I kept it with a collection in case of we want to have a separate list only for this. 
Anyway, fine with me to change it to int.